featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1238,8 +1238,8 @@ def test_cluster_creation(self, monkeypatch):
                             get_mock_client_cluster)
         try:
             cpus = len(psutil.Process().cpu_affinity())
-        except AttributeError:
-            cpus = psutil.cpu_count()
+        except AttributeError: # pragma: no cover
+            cpus = psutil.cpu_count() # pragma: no cover
```

Comment:
I believe you can put `# pragma: no cover` at the top of any block and it prevents tracking coverage for all lines in that block.  So you wouldn't need the pragma on line 1242.  I think this also works for functions.

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/1829/commits/4492edc60cf14e1ef5e72a05a18b9ddebf3ce390